pkg/server/routes.go

```diff
@@ -100,13 +102,28 @@ func (s *Server) routes(
 		}
 	}
 
-	var cedarLDAPClient cedarldap.Client
-	cedarLDAPClient = cedarldap.NewTranslatedClient(
-		s.Config.GetString(appconfig.CEDARAPIURL),
-		s.Config.GetString(appconfig.CEDARAPIKey),
-	)
+	//TODO: update this to have OKTA API live in it's own package?
+	var userSearchClient usersearch.Client
 	if s.environment.Local() || s.environment.Test() {
-		cedarLDAPClient = local.NewCedarLdapClient(s.logger)
+		userSearchClient = local.NewCedarLdapClient(s.logger)
```

Comment:
Maybe this would be best done at a later date (not in this PR) but since we're now abstracting the behavior of LDAP/Okta to just be a "User Search Client" (which I think is smart) we could probably rename this to something like `LocalUserSearchClient` or `MockUserSearchClient` and move it to `pkg/userserach`, since it's just a generic implementation of a search client that satisfies the interface

Food for thought - doesn't need to happen here, but wanted to get that thought/context shared!

Reply:
I will add this to the follow up Ticket